package.json

```diff
@@ -1,326 +1,153 @@
-    },
-    "devDependencies": {
-        "@types/semver": "^7.1.0",
-        "@types/glob": "^7.1.1",
-        "@types/jquery": "^3.3.30",
-        "@types/markdown-it": "0.0.9",
-        "@types/mocha": "^7.0.2",
-        "@types/node": "^13.13.2",
-        "@types/sarif": "^2.1.2",
-        "@types/vscode": "^1.44.0",
-        "mocha": "^7.1.2",
-        "tslint": "^6.1.1",
-        "tslint-microsoft-contrib": "^6.0.0",
-        "tslint-no-unused-expression-chai": "^0.1.4",
-        "typescript": "^3.8.3",
-        "vscode-test": "^1.0.2"
-    },
-    "dependencies": {
-        "semver": "^7.3.2",
-        "@microsoft/sarif-multitool": "^2.2.4",
-        "json-source-map": "^0.4.0",
-        "jsonpointer": "4.0.1",
-        "markdown-it": "^9.0.1",
-        "requirejs": "^2.3.6",
-        "vscode-nls": "^4.1.2",
-        "https-proxy-agent": "^5.0.0"
-    }
+	"name": "sarif-viewer",
+	"displayName": "Sarif Viewer",
+	"description": "The Sarif Viewer extension visualizes the results in a Sarif file",
+	"author": "Microsoft Corporation",
+	"license": "MIT",
+	"version": "2.16.0",
+	"publisher": "MS-SarifVSCode",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Microsoft/sarif-vscode-extension.git"
+	},
+	"bugs": {
+		"url": "https://github.com/Microsoft/sarif-vscode-extension/issues"
+	},
+	"icon": "icon.png",
+	"galleryBanner": {
+		"color": "#c9c2a0",
+		"theme": "light"
+	},
+	"categories": [
+		"Other"
+	],
+	"keywords": [
+		"sarif"
+	],
+	"engines": {
+		"vscode": "^1.44.0"
+	},
+	"activationEvents": [
+		"*"
+	],
+	"main": "./out/context.js",
+	"contributes": {
+		"configuration": {
+			"title": "Sarif Viewer",
+			"properties": {
+				"sarif-viewer.rootpaths": {
+					"type": "array",
+					"description": "%configuration.sarif-viewer.rootpaths.description%"
+				},
+				"sarif-viewer.explorer.openWhenNoResults": {
+					"description": "%sarif-viewer.explorer.openWhenNoResults.description%",
+					"type": "boolean",
+					"default": true
+				},
+				"sarif-viewer.updateChannel": {
+					"description": "%sarif-viewer.updateChannel.description%",
+					"type": "string",
+					"enum": [
+						"Default",
+						"Insiders"
+					],
+					"enumDescriptions": [
+						"%sarif-viewer.updateChannel.default.description%",
+						"%sarif-viewer.updateChannel.insiders.description%"
+					],
+					"default": "Default",
+					"scope": "application"
+				}
+			}
+		},
+		"languages": [
+			{
+				"id": "json",
+				"extensions": [
+					".sarif"
+				]
+			}
+		],
+		"commands": [
+			{
+				"command": "sarif.showPanel",
+				"title": "SARIF: Show Panel"
+			},
+			{
+				"command": "sarif.clearState",
+				"title": "SARIF: Clear State"
+			},
+			{
+				"command": "sarif.updateExtension",
+				"title": "SARIF: Update Extension"
+			},
+			{
+				"command": "sarif.apiOpenLogs",
+				"title": "SARIF: Test Open Logs via API"
+			},
+			{
+				"command": "sarif.apiCloseLogs",
+				"title": "SARIF: Test Close Logs via API"
```

Comment:
>SARIF:  [](start = 14, length = 7)

If you use category property, you shouldn't need to prefix everything with "SARIF:"

Reply:
Awesome. Updated. Anyway to not have to repeat the category property?

Reply:
Nope. An extension can have multiple categories. Sorry about that.